Source/FakeItEasy.Specs/DomainEvents.cs

```diff
@@ -1,8 +1,26 @@
 ﻿namespace FakeItEasy.Specs
 {
+    using System;

```

Comment:
I can go either way regarding reusing sample classes between test groups.
I suppose it opens the door for clashing tests (although all one has to do is add more tests that use the class until there's a problem and _then_ make a new one), but it also helps with what would otherwise be a proliferation of test classes.

As for the name, if the class _is_ to be shared, then naming becomes more difficult, as one has to keep renaming the class every time it's used for a new purpose. Me, I think the class names provide info about the intent and capabilities of the classes used therein, but the authors always think their makes sense!

That being said, other than a perhaps too-strong attachment to the phrase "RunsAmok" in the one class name, I'm not bothered at the prospect of adding extra classes, or renaming, if this is the popular opinion.